deployment/sandbox/flyte_generated.yaml

```diff
@@ -270,6 +270,21 @@ status:
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
+  name: flyteworkflows.flyte.lyft.com
+spec:
+  group: flyte.lyft.com
+  names:
+    kind: FlyteWorkflow
+    plural: flyteworkflows
+    shortNames:
+    - fly
+    singular: flyteworkflow
+  scope: Namespaced
+  version: v1alpha1
```

Comment:
Do we not need to use "versions" instead now?

Reply:
Oh I wasn't sure about this. I'll look it up!

Reply:
Fixed.

Reply:
Tested on a v1.22 k8s cluster:
```
> kubectl apply -f deployment/sandbox/flyte_generated.yaml
namespace/flyte created
namespace/kubernetes-dashboard created
namespace/projectcontour created
customresourcedefinition.apiextensions.k8s.io/extensionservices.projectcontour.io created
customresourcedefinition.apiextensions.k8s.io/flyteworkflows.flyte.lyft.com created
customresourcedefinition.apiextensions.k8s.io/httpproxies.projectcontour.io created
customresourcedefinition.apiextensions.k8s.io/tlscertificatedelegations.projectcontour.io created
serviceaccount/datacatalog created
serviceaccount/flyte-pod-webhook created
serviceaccount/flyteadmin created
serviceaccount/flytepropeller created
serviceaccount/kubernetes-dashboard created
serviceaccount/contour created
serviceaccount/contour-certgen created
serviceaccount/envoy created
role.rbac.authorization.k8s.io/kubernetes-dashboard created
role.rbac.authorization.k8s.io/contour-certgen created
clusterrole.rbac.authorization.k8s.io/flyte-pod-webhook created
clusterrole.rbac.authorization.k8s.io/flyteadmin created
clusterrole.rbac.authorization.k8s.io/contour created
clusterrole.rbac.authorization.k8s.io/flytepropeller created
clusterrole.rbac.authorization.k8s.io/kubernetes-dashboard created
rolebinding.rbac.authorization.k8s.io/kubernetes-dashboard created
rolebinding.rbac.authorization.k8s.io/contour created
clusterrolebinding.rbac.authorization.k8s.io/flyte-pod-webhook created
clusterrolebinding.rbac.authorization.k8s.io/flyteadmin-binding created
clusterrolebinding.rbac.authorization.k8s.io/flytepropeller created
clusterrolebinding.rbac.authorization.k8s.io/contour created
clusterrolebinding.rbac.authorization.k8s.io/kubernetes-dashboard created
clusterrolebinding.rbac.authorization.k8s.io/kubernetes-dashboard-admin created
configmap/clusterresource-template-dk5mbchdmt created
configmap/datacatalog-config-64k8dg9gck created
configmap/flyte-admin-config-dbg8dt2dgb created
configmap/flyte-console-config created
configmap/flyte-propeller-config-g52db28fmc created
configmap/flyte-scheduler-config created
configmap/kubernetes-dashboard-settings created
configmap/contour created
secret/db-pass-9dgchhk2bm created
secret/flyte-admin-auth created
secret/flyte-pod-webhook created
secret/flyte-secret-auth created
secret/user-info created
secret/kubernetes-dashboard-csrf created
secret/kubernetes-dashboard-key-holder created
service/datacatalog created
service/flyte-pod-webhook created
service/flyteadmin created
service/flyteconsole created
service/minio created
service/minio-direct created
service/postgres created
service/postgres-direct created
service/dashboard-metrics-scraper created
service/kubernetes-dashboard created
service/contour created
service/envoy created
deployment.apps/datacatalog created
deployment.apps/flyte-pod-webhook created
deployment.apps/flyteadmin created
deployment.apps/flyteconsole created
deployment.apps/flytepropeller created
deployment.apps/flytescheduler created
deployment.apps/minio created
deployment.apps/postgres created
Warning: spec.template.metadata.annotations[seccomp.security.alpha.kubernetes.io/pod]: deprecated since v1.19; use the "seccompProfile" field instead
deployment.apps/dashboard-metrics-scraper created
deployment.apps/kubernetes-dashboard created
deployment.apps/contour created
Warning: batch/v1beta1 CronJob is deprecated in v1.21+, unavailable in v1.25+; use batch/v1 CronJob
cronjob.batch/syncresources created
daemonset.apps/envoy created
job.batch/contour-certgen-v1.13.1 created
ingress.networking.k8s.io/flytesystem created
ingress.networking.k8s.io/minio created
```
```
> kubectl get po
NAME                                 READY   STATUS      RESTARTS   AGE
flytepropeller-7977844599-c64x6      1/1     Running     0          55s
flyte-pod-webhook-57676c4995-74k82   1/1     Running     0          55s
minio-6d8d486f7f-mktss               1/1     Running     0          55s
postgres-7d9cb5b958-rksqd            1/1     Running     0          54s
flyteconsole-78df95c765-8jvb8        1/1     Running     0          55s
flyteadmin-954fc55d4-c8rc9           3/3     Running     0          55s
datacatalog-5b49895855-d9r68         1/1     Running     0          55s
syncresources-27450381--1-frlc8      0/1     Completed   0          19s
flytescheduler-6fccd8b96-kvmns       1/1     Running     0          55s
```